madhav/src/main/java/com/lftechnology/phpjava/numbers/Fibonacci.java

```diff
@@ -0,0 +1,26 @@
+package main.java.com.lftechnology.phpjava.numbers;
+
+import main.java.com.lftechnology.phpjava.main.Scanner;
+
+public class Fibonacci {
+    
+    /**
+     * Displays sum of all even fibonacci numbers within given range
+     */
+    public void getFibonacciSum() {
+        System.out.println("Enter the limit:");
+        int max = Scanner.get().nextInt();
+        int f = 1;
+        int sum = 0;
+        int i = 0;
+        while(f + i < max) {
+            int j = f;
+            f = i + f;
+            i = j;
+            if(f % 2 == 0) {
+                sum += f;
+            }
+        } 
+        System.out.println("Sum of even fibonaccis upto " + max + ": " + sum);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS106)